patches/chromium/fix_remove_caption-removing_style_call.patch

```diff
@@ -0,0 +1,35 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Raymond Zhao <7199958+rzhao271@users.noreply.github.com>
+Date: Wed, 17 Aug 2022 13:49:40 -0700
+Subject: fix: Remove caption-removing style call
+
+There is a SetWindowLong call that removes WS_CAPTION for frameless
+windows, but Electron uses WS_CAPTION even for frameless windows.
+
+Removing this call only affects frameless windows, and it fixes
+a visual glitch where they showed a Windows 7 style frame
+during startup.
+
+The if statement was originally introduced by
+https://codereview.chromium.org/9372053/, and it was there to fix
+a visual glitch with the close button showing up during startup
+or resizing, but Electron does not seem to run into that issue
+for frameless windows even with that block commented out.
+
+diff --git a/ui/views/win/hwnd_message_handler.cc b/ui/views/win/hwnd_message_handler.cc
+index c6810f162806b36494885b2f63982a756d4dcd38..92d030a3d24a7bb282f698a5591cf33ae14e06ed 100644
+--- a/ui/views/win/hwnd_message_handler.cc
++++ b/ui/views/win/hwnd_message_handler.cc
+@@ -1723,12 +1723,6 @@ LRESULT HWNDMessageHandler::OnCreate(CREATESTRUCT* create_struct) {
+   SendMessage(hwnd(), WM_CHANGEUISTATE, MAKELPARAM(UIS_CLEAR, UISF_HIDEFOCUS),
+               0);
+ 
+-  if (!delegate_->HasFrame()) {
```

Comment:
Wrap around `#if 0` instead of deletion.

Only other places this gets exercised is for Chrome extension app windows https://source.chromium.org/chromium/chromium/src/+/main:extensions/components/native_app_window/native_app_window_views.cc;l=37 and PIP, both of which shouldn't get affected by this.